src/pages/ConnectedApps/ConnectedApps.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="container mt-0 mx-auto">
+    <div class="title mt-6 mb-11 px-7">
+      CONNECTED APPS
+    </div>
+    <div>
+      <div
+        v-for="app in apps"
```

Comment:
```suggestion
        v-for="(app, i) in apps"
        :key="i"
```
the key for loops should preferably be a number instead of a string